packages/core/src/sync/index.ts

```diff
@@ -784,11 +875,6 @@ export const createSync = async (args: CreateSyncParameters): Promise<Sync> => {
         historicalSync.kill();
         promises.push(realtimeSync.kill());
       }
-
-      eventQueue.pause();
-      eventQueue.clear();
-      promises.push(eventQueue.onIdle());
```

Comment:
Why'd you remove these?

Reply:
The `sync` module no longer contains a queue. Instead, it is synchronous with `sync-realtime`. This is primarily to maintain consistency for factory addresses. Without this change, it would be almost impossible to maintain `unfinalizedChildAddresses` and pass the correct value for the correct block height to `buildEvents`.